legend-engine-pure-code-compiled-core/src/main/resources/core/pure/executionPlan/javaPlatform/executionNode/graphFetch/inMemory/graphFetchInMemory.pure

```diff
@@ -48,7 +48,7 @@ function meta::pure::executionPlan::engine::java::graphFetch::inMemory::prepareF
    let currentNodeInfo = $node->match([
       r : InMemoryRootGraphFetchExecutionNode[1]        | let childNodeInfo   = $context->nodeInfosForPath($path + '.n1')->toOne();
                                                           let returnType      = if($r.checked, | ^CheckedJavaType(checkedOf = $simpleType, source = $childNodeInfo.returnType), | $simpleType);
-                                                          ^NodeInfo(path = $path, returnType = $returnType, graphFetchTrees = $childNodeInfo.graphFetchTrees);,
+                                                          ^NodeInfo(path = $path, returnType = $returnType, graphFetchTrees = $childNodeInfo.graphFetchTrees,data = $node.graphFetchTree);,
```

Comment:
Lets add a comment that this data field is deprecated and should not be used

Reply:
thank you sure